chart/epinio/Chart.yaml

```diff
@@ -12,4 +19,4 @@ maintainers:
 name: epinio
 sources:
 - https://github.com/epinio/epinio
-version: 0.3.0
+version: 0.4.0
```

Comment:
Done by `updatecli`, has to be removed. Sorry my first thought was false...

Reply:
done